frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -289,86 +301,112 @@ const Map: React.FC<MapProps> = ({
     }
   };
 
+  const [infoOpen, setInfoOpen] = React.useState(false);
+  const [layersOpen, setLayersOpen] = React.useState(false);
+
   return (
-    <ReactResizeDetector handleWidth>
-      {({ width }: any) => {
-        setMapWidth(width);
-        return (
-          <Container fluid className={classNames('px-0 map', { narrow: sidebarSize === 'narrow' })}>
-            {!disableMapFilterBar ? (
-              <Container fluid className="px-0 map-filter-container">
-                <Container className="px-0">
-                  <PropertyFilter
-                    defaultFilter={defaultFilterValues}
-                    agencyLookupCodes={agencies}
-                    adminAreaLookupCodes={administrativeAreas}
-                    propertyClassifications={propertyClassifications}
-                    onChange={handleMapFilterChange}
-                  />
-                </Container>
-              </Container>
-            ) : null}
-            <Row noGutters>
-              <Col>
-                {baseLayers?.length > 0 && (
-                  <BasemapToggle baseLayers={baseLayers} onToggle={handleBasemapToggle} />
-                )}
-                <ReactLeafletMap
-                  ref={mapRef}
-                  center={[lat, lng]}
-                  zoom={lastZoom}
-                  whenReady={() => {
-                    fitMapBounds();
-                  }}
-                  onclick={showLocationDetails}
-                  closePopupOnClick={interactive}
-                  onzoomend={e => setZoom(e.sourceTarget.getZoom())}
-                  onmoveend={handleBounds}
-                >
-                  {activeBasemap && (
-                    <TileLayer
-                      attribution={activeBasemap.attribution}
-                      url={activeBasemap.url}
-                      zIndex={0}
+    <PropertyPopUpContextProvider>
```

Comment:
You should only be wrapping the map with the provider, the filter does not need access to the popup context